tidb-cloud/integrate-tidbcloud-with-netlify.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Integrate TiDB Cloud with Netlify
+summary: Learn how to connect your TiDB Cloud clusters to Netlify projects.
+---
+
+# Integrate TiDB Cloud with Netlify
+
+[Netlify](https://netlify.com/) is an all-in-one platform for automating modern web projects. It replaces your hosting infrastructure, continuous integration, and deployment pipeline with a single workflow and integrates dynamic functionality like serverless functions, user authentication, and form handling as your projects grow.
+
+This guide describes how to connect your TiDB Cloud clusters to Netlify projects.
+
+## Prerequisites
+
+Before connecting, make sure the following prerequisites are met.
+
+### A Netlify account and a deployed site
+
+You are expected to have an account and a site in Netlify. If you do not have any, refer to the following links to create one:
+
+* [Sign up a new account](https://app.netlify.com/signup).
+* [Add a site](https://docs.netlify.com/welcome/add-new-site/) in Netlify. If you do not have an application to deploy, you can use the [TiDB Cloud Starter Template](https://github.com/tidbcloud/nextjs-prisma-example#deploy-on-netlify) to have a try.
+
+### A TiDB Cloud account and a TiDB cluster
+
+You are expected to have an account and a cluster in TiDB Cloud. If you do not have any, refer to [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md).
+
+One TiDB Cloud cluster can connect to multiple Netlify sites.
+
+### All IP addresses allowed for traffic filter in TiDB Cloud
+
+For Dedicated Tier clusters, make sure that the traffic filter of the cluster allows all IP addresses (set to `0.0.0.0/0`) for connection, this is because Netlify deployments use dynamic IP addresses.
+
+Serverless Tier clusters allow all IP addresses for connection by default, so you do not need to configure any traffic filter.
+
+## Connect via manually setting environment variables
+
+To use this method, make sure that your TiDB Cloud cluster [allows all IP addresses access](#all-ip-addresses-allowed-for-traffic-filter-in-tidb-cloud) and save your root password in the [Security Settings](/tidb-cloud/configure-security-settings.md) dialog.
```

Comment:
_root password_ should also be put in the *prerequisites* section. WDYT?

Reply:
I don't think _root password_  is a part of _prerequisites_. We could consider putting _root password_ in the following steps.

Reply:
Sounds good. Please make it a step. Otherwise, it might be ignored.